<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellViewer</class>
 <widget class="QWidget" name="CellViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Cell Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="thresholdLabel">
     <property name="text">
      <string>Treshold :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="thresholdBox">
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="valueDisplayLabel">
     <property name="text">
      <string>0xfff</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="valueLabel">
     <property name="text">
      <string>Value :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="releaseLabel">
     <property name="text">
      <string>Release (ms) :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="releaseBox">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
